### PR TITLE
Re-implement getVersion function in Ballerina

### PR DESCRIPTION
**Pull request by MadhukaHarith92**
## Purpose
Fixes https://github.com/ballerina-platform/ballerina-standard-library/issues/2728
## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests
- [ ] Updated the spec

### CODE REVIEW

File: ballerina/uuid.bal [hunk 1]
```diff
@@ -342,3 +344,43 @@ public isolated function toRecord(string|byte[] uuid) returns Uuid|Error {
     };
     return uuidRecord;
 }
+
+# Represents a UUID instance.
+class UUID {
```
Comment (daneshk) [inline ballerina/uuid.bal:349]: is it correct to name this as `UUID` as it only contains the least and most significant bits?

File: ballerina/uuid.bal [hunk 2]
```diff
@@ -342,3 +344,43 @@ public isolated function toRecord(string|byte[] uuid) returns Uuid|Error {
...
+    mostSigBits <<= 16;
+    mostSigBits |= u.timeHiAndVersion & 0xffff;
+
+    int leastSigBits;
+    string clockSeq = regex:split(name, "-")[3];    
+    int|error clockSeqInt = ints:fromHexString(clockSeq);
+    if clockSeqInt is int {
+        leastSigBits = clockSeqInt & 0xffff;
+    } else {
+        return error Error("Failed to get clockSeq value of the uuid");
```
Comment (daneshk) [inline ballerina/uuid.bal:379]:
```suggestion
        return error Error("Failed to get clock sequence value of the UUID");
```

File: ballerina/uuid.bal (hunk 1 above)
Comment (MadhukaHarith92) [reply ballerina/uuid.bal:349]: A `UUID` which is a 128-bit label is comprised with `leastSigBits` and `mostSigBits` which are 64 bit each. The other fields like `timeLow`, `timeMid`, etc. are derived from `leastSigBits` and `mostSigBits`. Therefore, I think it's okay to name it `UUID`.

These 2 are the only fields in Java's UUID class[[1]](https://docs.oracle.com/javase/7/docs/api/java/util/UUID.html) as well.

Another option is to name it something like `UuidMeta`. WDYT?

[1] https://docs.oracle.com/javase/7/docs/api/java/util/UUID.html

Comment (MadhukaHarith92) [reply ballerina/uuid.bal]: As discussed, removed the class in [d5f025a](https://github.com/ballerina-platform/module-ballerina-uuid/pull/374/commits/d5f025a7e5d2644bceb1b97861a46a273cc6ca51)

File: ballerina/Ballerina.toml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 [package]
 org = "ballerina"
 name = "uuid"
-version = "1.2.1"
+version = "1.2.2"
```
Comment (daneshk) [inline ballerina/Ballerina.toml:4]: This should be the next minor release right?

Comment (MadhukaHarith92) [reply ballerina/Ballerina.toml]: Updated to `1.3.0`

**Review by daneshk — APPROVED** [review]

**Pull request merged**